### PR TITLE
Update Beta to 1.3-rc1

### PR DESCRIPTION
**Pull request by BryanQuigley**
Refreshes patches and bumps depends as well

### CODE REVIEW

File: net.supertuxkart.SuperTuxKart.json [hunk 1]
```diff
@@ -1,17 +1,22 @@
 {
     "app-id": "net.supertuxkart.SuperTuxKart",
-    "default-branch": "beta",
+    "default-branch": "stable",
```
Comment (AsciiWolf) [inline net.supertuxkart.SuperTuxKart.json:3]: Shouldn't this be set to "beta"?

**Comment by AsciiWolf** [comment]
Thanks! I will test and merge this tomorrow. :-)

File: net.supertuxkart.SuperTuxKart.json (hunk 1 above)
Comment (BryanQuigley) [reply net.supertuxkart.SuperTuxKart.json]: Oops, yes.  fixed in new push.

**Comment by AsciiWolf** [comment]
And it's stable: https://twitter.com/supertuxkart/status/1442902924045144065 :-)

**Pull request merged**